Automattic/jetpack
# AI Assistant: introduce block Message component

Fixes https://github.com/Automattic/jetpack/issues/31105
Fixes https://github.com/Automattic/jetpack/issues/31244
Fixes https://github.com/Automattic/jetpack/issues/31106

## Proposed changes:

* AI Assistant: introduce block Message area

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Use the AI Assistant
* Check when the content is generated, confirm you see the message:

<img width="696" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/0e57b6d4-d278-4dcd-9d4d-9462e643ff9b">
